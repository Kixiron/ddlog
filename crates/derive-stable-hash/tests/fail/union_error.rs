use derive_stable_hash::StableHash;

#[derive(StableHash)]
union Foo {
    bar: u32,
    baz: usize,
    bing: u8,
}

fn main() {}
