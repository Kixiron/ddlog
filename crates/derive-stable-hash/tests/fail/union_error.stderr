error: Deriving StableHash is only supported for structs and enums, not unions
 --> tests/fail/union_error.rs:4:1
  |
4 | union Foo {
  | ^^^^^
